server/src/main/kotlin/suwayomi/tachidesk/graphql/mutations/MangaMutation.kt

```diff
@@ -71,7 +82,7 @@ class MangaMutation {
         )
     }
 
-    fun updateMangas(input: UpdateMangasInput): UpdateMangasPayload {
+    suspend fun updateMangas(input: UpdateMangasInput): UpdateMangasPayload {
```

Comment:
Public functions in the `mutations` and `queries` packages cannot be suspend, but they can return a `CompleteableFuture` created by the `future {}` function

Reply:
fixed